### PR TITLE
feat category

### PR DESCRIPTION
**Pull request by AmishMishra11**
image

### CODE REVIEW

**Review by SukanyaSen530 — APPROVED** [review]

File: category/category.html [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+                <h2>Friends Quiz</h2>
+                <p>Take a quiz to test yourself, contains 5 questions.</p>
+              </div>
+            </a>
+          </button>
+        </div>
+        <div class="cat cat-1">
+          <button>
+            <a href="#">
+              <img src="/images/friend.jpg" alt="category 1" />
```
Comment (Prabhav-Sharma) [inline category/category.html]: Change alt text

**Review by amarphule — CHANGES_REQUESTED** [review]
You can check the contrast ratio of header
am

File: category/category.html (hunk 1 above)
Comment (AmishMishra11) [reply category/category.html]: ive renamed it

**Comment by AmishMishra11** [comment]
> You can check the contrast ratio of header am

Thank you for pointing it out I've fixed contrast ratio

**Pull request merged**